src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -44,21 +44,22 @@ const CategoryDropdown = ({
   const dispatch = useAppDispatch()
   const { openModal, closeModal } = useModalContext()
 
-  const handleResponseError = (error?: ErrorResponse) => {
-    dispatch(
-      openAlert({
-        severity: snackbarVariants.error,
-        message: getErrorKey(error)
-      })
-    )
-  }
+  const handleResponseError = useCallback(
+    (error?: ErrorResponse) => {
```

Comment:
Please, change it to the `ResponseError` because only `ResponseError` will be thrown by `baseService`
```suggestion
    (error: ResponseError) => {
```

Reply:
Not resolved, please remove `?`